README.md

```diff
@@ -1,23 +1,31 @@
-# Haikuporter
+# haikuporter
 
-The HaikuPorter tool is provided to ease the fetching, patching and building of source code. It can be compared to a slim version of [Gentoo Portage](https://www.gentoo.org/main/en/about.xml). Each port contains the [Haiku](http://haiku-os.org) specific patches to the original source code. It fetches the original source code, applies the Haiku-specific patches, builds the software, and packages it.
```

Comment:
this part of the README seems quite important, explaining what the tool does.

On the other hand I don't think the collection of "shields" is really that useful.

Reply:
I agree - this is definitely more useful than shields. I'm changing this.

Reply:
Done